content/docs/data/Import.md

```diff
@@ -0,0 +1,319 @@
+-   It can keep the Elasticsearch index synced in realtime with the data source.
+    > Note: This is currently only supported for MongoDB and Postgres.
+-   `abc import` is a single line command that allows doing all of the above. It doesn’t require any external dependencies, takes zero lines of code configuration, and runs as an isolated process with a minimal resource footprint.
+
+### Installation
+
+> Command Line: `docker pull appbaseio/abc`
+
+You can also download from [GitHub](https://github.com/appbaseio/abc/releases/latest) or get the [Docker image](https://hub.docker.com/r/appbaseio/abc/)
+
+It is possible to import data from various database sources. See the API below to import from the database that suits your need.
+
+###Postgres
+
+```bash
+abc import --src_type=postgres --src_uri=<uri> <elasticsearch_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-data-from-postgres-to-elasticsearch-6eebc5cc0f0f)
+
+###MongoDB
+
+```bash
+abc import --src_type=mongodb --src_uri=<uri> <elasticsearch_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-data-from-mongodb-to-elasticsearch-ee5a74695945)
+
+###MySQL
+
+```bash
+abc import --src_type=mysql --src_uri=<uri> <elasticsearch_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-data-from-mysql-to-elasticsearch-b59289e5025d)
+
+###Microsoft SQL Server
+
+```bash
+abc import --src_type=mssql --src_uri=<uri> <elasticsearch_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-data-from-mssql-to-elasticsearch-341963a054dd)
+
+###Elasticsearch
+
+```bash
+abc import --src_type=elasticsearch --src_uri=<elasticsearch_src_uri> <elasticsearch_dest_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-data-from-elasticsearch-to-elasticsearch-301c7a243c84)
+
+###JSON
+
+```bash
+abc import --src_type=json --src_uri=<uri> --typename=<target_type_name> <elasticsearch_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-data-from-json-to-elasticsearch-92f582c53df4)
+
+###CSV
+
+```bash
+abc import --src_type=csv --src_uri=<uri> --typename=<target_type_name> <elasticsearch_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-data-from-csv-to-elasticsearch-17d290a5974f)
+
+###Firestore
+
+```bash
+abc import --src_type=firestore --sac_path=<path_to_service_account_credentials> <elasticsearch_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-from-firestore-to-elasticsearch-80286fc8e58b)
+
+---
+
+##Rest API
+This is just getting started with Rest API. You can read more about it over [here](https://rest.appbase.io/).
+
+### Setup
```

Comment:
@ShahAnuj2610 This section should talk about the indexing part only. It should mention the use of PUT / POST endpoint for creating / updating a document and the bulk API endpoint and how to use them.